packages/orchestrator-ui-components/src/hooks/ProcessesHooks/useMutateProcesses.ts

```diff
@@ -0,0 +1,51 @@
+import { useContext } from 'react';
+import { useMutation, useQueryClient } from 'react-query';
+
+import { signOut } from 'next-auth/react';
+
+import { OrchestratorConfigContext } from '../../contexts';
+import { useSessionWithToken } from '../useSessionWithToken';
+
+export const useMutateProcesses = () => {
+    const { processesEndpoint } = useContext(OrchestratorConfigContext);
+    const { session } = useSessionWithToken();
+    const queryClient = useQueryClient();
+
+    const genericRequestHeaders = {
+        authorization: session ? `Bearer ${session.accessToken}` : '',
+    };
+
+    const retryAllProcessesServiceCall = async () =>
+        await fetch(`${processesEndpoint}/resume-all`, {
+            method: 'PUT',
+            headers: {
+                ...genericRequestHeaders,
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({}),
```

Comment:
Needed?

Reply:
This is maybe a bit extreme indeed: changing it into "{}" because that is what the backend expects